Fix implicit declaration of functions and too few arguments to function call.
--- configure.orig	2022-05-27 13:55:37.000000000 -0500
+++ configure	2022-06-24 03:54:02.000000000 -0500
@@ -1258,6 +1258,7 @@
 #line 1261 "configure"
 #include "confdefs.h"
 #include <ctype.h>
+#include <stdlib.h>
 #define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 #define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
 #define XOR(e, f) (((e) && !(f)) || (!(e) && (f)))
@@ -2318,6 +2319,7 @@
 #line 2321 "configure"
 #include "confdefs.h"
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/types.h>
 main()
 {
@@ -2358,6 +2360,7 @@
 #line 2361 "configure"
 #include "confdefs.h"
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/types.h>
 main()
 {
@@ -2398,6 +2401,7 @@
 #line 2401 "configure"
 #include "confdefs.h"
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/types.h>
 main()
 {
@@ -2439,6 +2443,7 @@
 #line 2442 "configure"
 #include "confdefs.h"
 #include <stdio.h>
+#include <stdlib.h>
 #include <sys/types.h>
 main()
 {
@@ -8941,8 +8946,9 @@
 #line 8940 "configure"
 #include "confdefs.h"
 
+#include <X11/Intrinsic.h>
 int main() {
-${x_direct_test_function}()
+${x_direct_test_function}(0)
 ; return 0; }
 EOF
 if { (eval echo configure:8947: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
